ab2d-fhir/src/main/java/gov/cms/ab2d/fhir/Versions.java

```diff
@@ -82,13 +82,8 @@ static Object getObject(FhirVersion version, String className) {
                 Class clazz;
                 try {
                     clazz = Class.forName(fullName);
-                } catch (ClassNotFoundException e) {
-                    log.error("Unable to create class " + fullName);
-                    return null;
-                }
-                try {
                     object = clazz.getDeclaredConstructor(null).newInstance();
-                } catch (InstantiationException | IllegalAccessException | InvocationTargetException | NoSuchMethodException e) {
```

Comment:
This is what I was talking about in our DM chat

Reply:
I might need an explanation of what is going on here. Nothing to do with what you removed, but...this code block in general is quite confusing.